### ISSUE TITLE
Why can there be 4 different breakends for a DUP?

### ISSUE THREAD

**Issue by weishwu** [PapenfussLab/gridss#383]
Hi, I've run GRIDSS and the simple sv annotator. For most of the SVs there are two lines in the VCF that contain the same pair of breakends but in different orientation (one has A and B and the other has B and A). But some of them have different breakends, e.g.
```
11      57655541        gridss38_80026o A       ]11:57655591]A  1458.93 PASS    AS=1;ASQ=434.84;ASRP=0;ASSR=42;BA=0;BANRP=0;BANRPQ=0;BANSR=0;BANSRQ=0;BAQ=0;BASRP=0;BASSR=0;BEID=asm38-224743,asm38-65160,asm38-65163;BQ=42.06;BSC=2;BSCQ=42.06;BUM=0;BUMQ=0;BVF=0;CAS=2;CASQ=103.58;CIPOS=0,11;CIRPOS=0,11;CQ=1257.32;EVENT=gridss38_80026;HOMLEN=11;HOMSEQ=AGCTGCCCCTC;IC=0;IHOMPOS=-13,20;IQ=0;PARID=gridss38_80026h;RAS=2;RASQ=527.73;REF=68;REFPAIR=29;RP=0;RPQ=0;SB=0.3448276;SC=1X10N1X99M;SR=15;SRQ=392.78;SVLEN=60;SVTYPE=BND;VF=17;SIMPLE_TYPE=DUP    GT:ASQ:ASRP:ASSR:BANRP:BANRPQ:BANSR:BANSRQ:BAQ:BASRP:BASSR:BQ:BSC:BSCQ:BUM:BUMQ:BVF:CASQ:IC:IQ:QUAL:RASQ:REF:REFPAIR:RP:RPQ:SR:SRQ:VF   .:0:0:0:0:0:0:0:0:0:0:0:0:0:0:0:0:0:0:0:0:0:47:29:0:0:0:0:0     .:434.84:0:42:0:0:0:0:0:0:0:42.06:2:42.06:0:0:0:103.58:0:0:1458.93:527.73:21:0:0:0:15:392.78:17
11      57655602        gridss38_80026h C       C[11:57655552[  1329.2  PASS    AS=2;ASQ=527.73;ASRP=0;ASSR=37;BA=0;BANRP=0;BANRPQ=0;BANSR=0;BANSRQ=0;BAQ=0;BASRP=0;BASSR=0;BEID=asm38-224743,asm38-65160,asm38-65163;BQ=187.74;BSC=7;BSCQ=162.8;BUM=1;BUMQ=24.95;BVF=2;CAS=2;CASQ=103.58;CIPOS=-11,0;CIRPOS=-11,0;CQ=1257.32;EVENT=gridss38_80026;HOMLEN=11;HOMSEQ=AGCTGCCCCTC;IC=0;IHOMPOS=-24,9;IQ=0;PARID=gridss38_80026o;RAS=1;RASQ=305.11;REF=89;REFPAIR=32;RP=0;RPQ=0;SB=0.35869566;SC=89M1X10N1X;SR=15;SRQ=392.78;SVLEN=60;SVTYPE=BND;VF=16;SIMPLE_TYPE=DUP     GT:ASQ:ASRP:ASSR:BANRP:BANRPQ:BANSR:BANSRQ:BAQ:BASRP:BASSR:BQ:BSC:BSCQ:BUM:BUMQ:BVF:CASQ:IC:IQ:QUAL:RASQ:REF:REFPAIR:RP:RPQ:SR:SRQ:VF   .:0:0:0:0:0:0:0:0:0:0:52.79:1:27.84:1:24.95:2:0:0:0:0:0:45:32:0:0:0:0:0 .:527.73:0:37:0:0:0:0:0:0:0:134.95:6:134.95:0:0:0:103.58:0:0:1329.2:305.11:44:0:0:0:15:392.78:16
```

These two lines have 4 breakends 57655541 (A), 5765552 (B), 57655591 (C) and 57655602 (D). The first line is A->C and the second is B->D. The distance between A and B equals to the distance between C and D. And they have matched variant ID: gridss38_80026o and gridss38_80026h (By the way what do "o" and "h" mean?), and their svtype is DUP. Why are there 2 pairs of breakends in this DUP while most of other DUPs have only one pair of breakends? 

Thanks.

**Comment by d-cameron** [comment]
This is a single DUP call. "o" and "h" are the l<ins>o</ins>w and <ins>h</ins>igh breakends that compose a breakpoint.

The problem is there is an 11bp microhomology and the two sides are reporting different nominal positions so they don't match up. This bug was fixed in GRIDSS version 2.9.0

**Comment by d-cameron** [comment]
Just to clarify: these two breakends form a single breakpoint (hence the matching IDs) and both interpretations of the call are equivalent (see Figure 9 in the VCF specifications for an example).

The QUAL scores being different is another bug which has also fixed.
